src/components/core/pivotTable/agGridDataSource.ts

```diff
@@ -30,6 +30,15 @@ export const areTotalsChanged = (gridApi: GridApi, newTotals: IGridTotalsRow[])
     return false;
 };
 
+const isInvalidGetRowsRequest = (startRow: number, gridApi: GridApi): boolean => {
+    const bottomRowIndex = ApiWrapper.getPaginationBottomRowIndex(gridApi);
+    if (typeof bottomRowIndex === "number") {
```

Comment:
How about `if (bottomRowIndex !== null)`? `null` is return value of `ApiWrapper.getPaginationBottomRowIndex()` saying that something is wrong.